src/api/public/apidocs-new/OBS-v2.10.50.yaml

```diff
@@ -387,6 +387,8 @@ paths:
     $ref: 'paths/source_project_name_package_name_binary_filename_attribute.yaml'
 
   # Staging Workflow
+  /staging/{staging_workflow_project}/backlog:
```

Comment:
I believe `staging_workflow_project` should be `project_name` to use the parameter defined below

Reply:
Indeed. I should use `project_name` to define all the staging-related endpoints. The filenames are going to change as well. I move this PR to draft until I adapt everything.